docs/data/external-oasis-repos.md

```diff
@@ -123,16 +123,15 @@ Verify that the repository is readable over HTTP:
 
 That should print several lines including some gibberish at the end.
 
-Host a Repository on OASIS
-------------------------
+Hosting a Repository on OASIS
+-----------------------------
 
 1.  **Verify your VO's OIM registration is up-to-date**.  All repositories need to be associated with a VO; the VO
     needs to assign an _OASIS manager_ in OIM who would be responsible for the contents of any of the VO's repositories
     and will be contacted in case of issues. To designate an OASIS manager, have the VO manager update the
     [OIM registration](https://oim.grid.iu.edu).
 
-2.  The repository administrator should **create a [support ticket](https://ticket.opensciencegrid.org/goc/submit)**
-    using the following template:
+1.  Create a [support ticket](https://ticket.opensciencegrid.org/goc/submit) using the following template:
 
         Please add a new CVMFS repository to OASIS for VO %RED%voname%ENDCOLOR% using the URL 
             http://%RED%fully.qualified.domain%ENDCOLOR%:8000/cvmfs/%RED%example.opensciencegrid.org%ENDCOLOR%
```

Comment:
The URL to the instructions on the next line will now be incorrect.  Probably we should change it to the new URL so that the GOC staff will know immediately where to look for their instructions, although we could leave that out if they all knew where to look.